test/performance/odp_ipsec.c

```diff
@@ -693,11 +728,14 @@ run_measure_one_async(ipsec_args_t *cargs,
 		      time_record_t *start,
 		      time_record_t *end)
 {
+	int in_flight, pkts_allowed, i, num_pkts, rc = 0;
+	const int max_in_flight = cargs->in_flight;
+	const int burst_size = cargs->burst_size;
 	odp_ipsec_out_param_t param;
 	odp_pool_t pkt_pool;
 	odp_queue_t out_queue;
-	odp_packet_t pkt = ODP_PACKET_INVALID;
-	int rc = 0;
+	odp_packet_t pkt_out[POOL_NUM_PKT];
+	odp_packet_t pkt[POOL_NUM_PKT];
```

Comment:
These could also be smaller. If moved down in the loop, then num_pkts as the size would suffice, it seems.

Reply:
...or max_in_flight.

Reply:
Agreed. Will move it deeper and use max_in_flight & num_pkts.